pkg/controllers/provisioning/scheduling/scheduler.go

```diff
@@ -581,6 +589,11 @@ func (s *Scheduler) addToNewNodeClaim(ctx context.Context, pod *corev1.Pod) erro
 		if i >= idx {
 			return false
 		}
+
+		if minValuesAutoRelaxed {
```

Comment:
Do we think we need to expose this externally? If we do, does it make sense to put this as a property of the requirement vs. exposing it as a return value?